### PR TITLE
Rolex/Wimbledon ad only on desktops with a fixed height

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Pat asked that (1) the Rolex ad does not show up on tablets and (2) the top slot has a fixed height on desktops.

(1) can be achieved in targeting in DFP, but (2) can only be done via a media query. The only solution I found was to therefore delay the ad until the `leftCol` breakpoint has been crossed—i.e. screens smaller than 1140px wide won't see the ad, even on desktops.

I'm happy to improve if someone has a better idea.

cc @guardian/commercial-dev

### CODE REVIEW

**Comment by rich-nguyen** [comment]
👍  can't think of a better way :)

File: common/app/views/support/Commercial.scala [hunk 1]
```diff
@@ -33,28 +33,36 @@ object Commercial {
       metaData.id == "uk/technology"
     }
 
+    private def isWimbledonEnabled(metaData: MetaData, edition: Edition) = {
+      (metaData.id == "sport/tennis" || metaData.id == "sport/wimbledon") && edition.id == "UK"
```
Comment (kelvin-chappell) [inline common/app/views/support/Commercial.scala]: Better if edition check is `edition == Uk`

File: common/app/views/support/Commercial.scala [hunk 2]
```diff
@@ -33,28 +33,36 @@ object Commercial {
...
+      if (WimbledonTopAd.isSwitchedOn && isWimbledonEnabled(metaData, edition)) {
+          topSlotAdSizes + wimbledonLeftCol
+      } else {
+          topSlotAdSizes
+      }
     }
 
     // The sizesOverride parameter is for testing only.
     def cssClasses(metaData: MetaData, edition: Edition, sizesOverride: Seq[AdSize] = Nil): String = {
+      var wimbledonClasses = if (WimbledonTopAd.isSwitchedOn && isWimbledonEnabled(metaData, edition)) Seq("top-banner-ad-container--wimbledon") else Nil
```
Comment (kelvin-chappell) [inline common/app/views/support/Commercial.scala]: There only seems to be one css class here, so could be `wimbledonClass: String`.
And should be a `val`.

File: common/app/views/support/Commercial.scala [hunk 3]
```diff
@@ -33,28 +33,36 @@ object Commercial {
...
       val classes = Seq(
         "top-banner-ad-container",
         "top-banner-ad-container--desktop",
         "top-banner-ad-container--above-nav",
         "js-top-banner-above-nav",
         "top-banner-ad-container--reveal"
-      )
+      ) ++ wimbledonClasses
```
Comment (kelvin-chappell) [inline common/app/views/support/Commercial.scala]: But then this becomes less neat.  Would have to use a fold or map/getOrElse.  So feel free to ignore last comment.  Apart from val.

**Comment by kelvin-chappell** [comment]
👍

**Pull request merged**